lighthouse-core/report/report-generator.js

```diff
@@ -107,6 +99,20 @@ class ReportGenerator {
   }
 
   /**
+   * Generated time of report
+   * @return {string}
+   */
+  _generatedTime() {
+    const options = {
```

Comment:
recreating these static options and formatter instance seems unneeded. Consider pulling out into module global and private vars